scripts/build.sh

```diff
@@ -2,25 +2,52 @@
 
 set -e
 
-PATH=$(npm bin):"$PATH"
-export PATH
 CLEANSLATE="node_modules/cleanslate/docs/files/cleanslate.css"
 
+isWindowsMingw() {
+  local is_mingw="False"
+  if [ "$(uname | cut -c 1-5)" == "MINGW" ]; then
+    is_mingw="True"
+  fi
+
+  echo -n "${is_mingw}"
+}
+
+if [ "$(isWindowsMingw)" == "True" ]; then
+  NPM_BIN_DIR="$(cygpath $(npm bin))"
+  PATH=$NPM_BIN_DIR:$PATH
+else
+  PATH="$(npm bin):$PATH"
+fi
+
+export PATH
+
+mkdir -p build
+mkdir -p build/static
 mkdir -p generated/static
 mkdir -p generated/static/clippy
+
 scripts/excmds_macros.py
```

Comment:
This needs to be `py -3 scripts/excmds_macros.py`, else I get

```
olie@OLIE-PC MINGW64 ~/projects/tridactyl (gsbabil-master)
$ npm run build

> tridactyl@0.1.0 build C:\Users\olie\projects\tridactyl
> sh scripts/build.sh

/usr/bin/env: ‘python3’: No such file or directory
```

So we should just put in a little guard like you have done on line 16, or perhaps just `WINPYTHON="py -3"` in that if statement and then `$WINPYTHON scripts/excmds_macros.py`.